packages/contracts/src/dollar/core/ERC1155Ubiquity.sol

```diff
@@ -23,6 +23,22 @@ contract ERC1155Ubiquity is ERC1155, ERC1155Burnable, ERC1155Pausable {
     mapping(address => uint256[]) private _holderBalances;
     uint256 private _totalSupply;
```

Comment:
Can't we make it public to eliminate [such refactors](https://github.com/ubiquity/ubiquity-dollar/pull/681/files#diff-cae9a99180379f71b4ba2e7d815f51849fcdbe1b33748736aeb243cd046c491fR102)?

Reply:
I think it's ok to make it `public` or `internal` and to remove these methods: `accessTotalSupply(), _incrementTotalSupply(), _decrementTotalSupply()`

Reply:
> I think it's ok to make it `public` or `internal` and to remove these methods: `accessTotalSupply(), _incrementTotalSupply(), _decrementTotalSupply()`

sure yeah  that methods can be removed then imma do it